scripts/gen_ta_src.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python3
+import os
```

Comment:
This requires proper commit description and Readme file with the description how to use this

Reply:
Done